docker-entrypoint-initdb.d/001-init.sql

```diff
@@ -0,0 +1,57 @@
+-- MySQL dump 10.13  Distrib 8.4.0, for Linux (x86_64)
```

Comment:
Since we are using the serlo database , this should be deleted
https://github.com/serlo/api.serlo.org/pull/1665